ui/src/models/answer.js

```diff
@@ -15,9 +15,13 @@ export function answerQuestion(answer, questionId, token) {
 
 }
 
-export function getAnswersByQuestionId(id) {
+export function getAnswersByQuestionId(id, token) {
     const options = {
-        url: `/api/answers/question/${id}`
+        url: `/api/answers/question/${id}`,
+        method: 'GET',
+        headers: {
```

Comment:
why is the token included? auth should never be a concern for normal requests. The cookie will be sent by it self between the client and the server. This token represents the openId token and is only used during signin.

Reply:
So it can be removed from answerQuestion as well?

Reply:
I removed it and it still seems to work.